project/Settings.scala

```diff
@@ -29,14 +29,14 @@ object Settings {
 
     // ScalaJS libraries
     val scalaDom     = "0.9.1"
-    val scalajsReact = "0.11.2"
+    val scalajsReact = "0.11.3"
     val scalaCSS     = "0.5.0"
     val uPickle      = "0.4.0"
-    val booPickle    = "1.2.4"
-    val diode        = "1.0.0"
-    val javaTimeJS   = "0.1.0"
+    val booPickle    = "1.2.5"
+    val diode        = "1.1.0"
+    val javaTimeJS   = "0.2.0"
     val javaLogJS    = "0.1.0"
-    val scalaJQuery  = "1.0-RC6"
+    val scalaJQuery  = "1.0"
```

Comment:
In case I want to version bump myself in the future, how do you pick the versions? Just look at the latest ones and check that they build correctly together?